ibm_catalog.json

```diff
@@ -0,0 +1,216 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-ease",
+        "label": "Cloud automation for Enterprise Application Service",
+        "product_kind": "solution",
+        "tags": [
+          "ibm_created",
+          "target_terraform",
+          "terraform",
+          "solution"
+        ],
+        "keywords": [
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution"
+        ],
+        "short_description": "Creates and configures an instance of Enterprise Application Service for Java",
+        "long_description": "This architecture supports creating and configuring an instance of Enterprise Application Service for Java to build and deploy applications on public cloud.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-ease/blob/main/README.md",
+        "offering_icon_url": "",
```

Comment:
you need an icon or we cant publish

Reply:
added icon (URL referencing to main branch, so the URL will work when merged)